aws/resource_aws_api_gateway_method_test.go

```diff
@@ -207,14 +207,14 @@ func testAccCheckAWSAPIGatewayMethodAttributes(conf *apigateway.Method) resource
 		if val, ok := conf.RequestParameters["method.request.header.Content-Type"]; !ok {
 			return fmt.Errorf("missing Content-Type RequestParameters")
 		} else {
-			if *val != false {
+			if !*val {
```

Comment:
It looks like the logic got flipped here.

```
--- FAIL: TestAccAWSAPIGatewayMethod_basic (41.36s)
    testing.go:538: Step 0 error: Check failed: Check 2/5 error: wrong Content-Type RequestParameters value
--- FAIL: TestAccAWSAPIGatewayMethod_customrequestvalidator (34.10s)
    testing.go:538: Step 0 error: Check failed: Check 2/6 error: wrong Content-Type RequestParameters value
```

```suggestion
			if *val {
```